Auth0.Windows/Auth0Client.cs

```diff
@@ -199,10 +205,20 @@ public Task<JObject> GetDelegationToken(string targetClientId, IDictionary<strin
         /// <summary>
         /// Log a user out of a Auth0 application.
         /// </summary>
-        public void Logout()
+        /// <param name="forceClearCache">Force clearing the browser cache, no matter the login method</param>
+        public void Logout(bool forceClearCache = false)

```

Comment:
I would actually prefer not having the parameter altogether. Why not just check the `clearCache` field? Like this:

```
public void Logout()
        {
            this.CurrentUser = null;
            if(this.clearCache)
                WebBrowserHelpers.ClearCache();
        }
```